spec/util/java_version_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+require 'logstash/util/java_version'
+
+describe "LogStash::Util::JavaVersion" do
+  let(:mod) { LogStash::Util::JavaVersion }
+  it "should get the current java version if we're on Java" do
+    if RUBY_ENGINE == "jruby"
```

Comment:
you are testing java versions, so it makes no sense to just check here, you should check before running the test.